labgrid/resource/adb.py

```diff
@@ -0,0 +1,23 @@
+import attr
+
+from ..factory import target_factory
+from .common import NetworkResource, Resource
+
+
+@target_factory.reg_resource
+@attr.s(eq=False)
+class ADBDevice(Resource):
+    serial = attr.ib(validator=attr.validators.instance_of(str))
```

Comment:
Maybe change this to `serialno` to avoid confusion with `serial` devices?

Reply:
Changed